### PR TITLE
Backport link integrity improvements and References and Links menu to 16.x

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
/cc @plone/volto-team I will merge this in the upcoming days. If someone has any problem with it, please state it so we can discuss it. Please notice that this do not backport the relations control panel. This will remain to be a Volto > 16 feature.

Ordinarily, the current policy is not to backport to LTS any feature, but this one proved to be important enough to make it there. In fact, it was already implemented with a feature flag, so we could backport it to 16 at some point. It didn't happen until now. The feature flag will be opt-in in 16 so no disruption in sites will happen.

**Review by davisagli — APPROVED** [review]
@sneridagh Looks acceptable to me, since it's disabled by default in the backport.

I would recommend editing the towncrier entries to combine into a single entry that mentions the feature which is backported and how to enable it.

**Comment by sneridagh** [comment]
@davisagli done.

**Review by giuliaghisini — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -113,22 +113,6 @@ On Plone 5, Volto is currently tested with the following packages pinned to spec
 - plone.rest 2.0.0a5
 - plone.volto 4.0.0a13
 
-This would be the docker command to spawn a Plone 5 container with the right KGS versions:
-
-```shell
-docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e ADDONS="plone.restapi==8.30.0 plone.volto==4.0.0a13 plone.rest==2.0.0a5 plone.app.iterate==4.0.2 plone.app.vocabularies==4.3.0" -e PROFILES="plone.volto:default-homepage" plone/plone-backend
```
Comment (ksuess) [inline README.md:119]: plone.restapi==8.39.0
The relations service is introduced in 8.39.0
https://github.com/plone/plone.restapi/commit/880e98321b0120954c9b9ab88c9ea689df5ad0ce

**Review by ksuess — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sneridagh) [reply README.md:119]: @ksuess GH just showed me your comment :( I will update the README in other PR

**Pull request merged**